components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/Utils.java

```diff
@@ -106,4 +109,24 @@ public static ServiceAdmin getServiceAdmin(MessageContext messageContext) {
         }
         return serviceAdmin;
     }
+
+    /**
+     * Util method to return the specified  property from a properties file.
+     *
+     * @param srcFile - The source file which needs to be looked up.
+     * @param key     - Key of the property.
+     * @return - Value of the property.
+     */
+    public static String getProperty(File srcFile, String key) {
+
+        String value = null;
+        try (FileInputStream fis = new FileInputStream(srcFile)) {
+            Properties properties = new Properties();
+            properties.load(fis);
+            value = properties.getProperty(key);
+        } catch (Exception e) {
```

Comment:
Shall we not catch a genric exception. Also should we not throw these exceptions?

Reply:
Fixed in 8d41bd6044fc5c95f39f6bf345f0876a1e3947d0